deployments/cloudfoundry/buildpack/bin/supply

```diff
@@ -1,79 +1,86 @@
-#!/usr/bin/env python3
+#!/bin/bash
 
-import os
-import shutil
-from os import makedirs, environ as ENV
-from pathlib import Path
-from sys import argv as ARGV
-from urllib.request import build_opener, HTTPRedirectHandler
+LATEST_VERSION="latest"
+JAVA_AGENT_VERSION="$SPLUNK_OTEL_JAVA_VERSION"
+if [ "$JAVA_AGENT_VERSION" == "" ]; then
```

Comment:
I think it is more common to use something like `if [ "x$JAVA_AGENT_VERSION = "x" ]; then` or `if [ -z "$JAVA_AGENT_VERSION" ]; then` Idk, might be a bash vs sh thing.